finality-aleph/src/abft/mod.rs

```diff
@@ -84,7 +84,7 @@ impl<S: legacy_aleph_bft::Signature> legacy_aleph_bft::PartialMultisignature for
     }
 }
 
-impl<S: legacy_aleph_bft::Signature> current_aleph_bft::PartialMultisignature for SignatureSet<S> {
+impl<S: current_aleph_bft::Signature> current_aleph_bft::PartialMultisignature for SignatureSet<S> {
```

Comment:
Not sure what should be here. It also compiles with `legacy_aleph_bft::Signature` here, and in line 75 we can as well put `current_aleph_bft::Signature`. I'm guessing that they're just the same.

Reply:
They kinda are the same. I guess the compiler unwinds the below definition which is the same in legacy and current and doesn't distinguish between them.
```
pub trait Signature: Debug + Clone + Codec + Send + Sync + Eq + 'static {}
```